src/entity/Station.java

```diff
@@ -8,7 +8,7 @@ public class Station implements StationInterface{
     private final String parentLine;
     private final float latitude;
     private final float longitude;
-    private final List<String> amenitiesList;
+    private List<String> amenitiesList; //TODO: Due to this variable being assigned a valid value AFTER initial text file is read, we need to make this non-final. But this violates CAE?
```

Comment:
Any follow-up from instructors?

Reply:
I talked to the TA about this and they were ok with this being non-final (due to the fact that we were only changing amenitiesList) in the DAO (will elaborate on this more either here or during meetings)